themes/scss/infowindow/cartodb-infowindow-default.scss

```diff
@@ -5,312 +5,301 @@
 .cartodb-infowindow {
   position: absolute;
 }
-
-.Cdb-infowindow {
-  background: #fff;
-  border-radius: 4px;
+.CDB-infowindow {
   position: relative;
-  box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, .24);
-  font-family: 'Open Sans';
-  cursor: default;
   /* remove when fix the position */
   transform: translateY(-24px) translateX(4px);
+  border-radius: 4px;
+  background: #FFF;
+  font-family: 'Open Sans';
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.24);
+  cursor: default;
 }
-
-.Cdb-infowindow-container {
+.CDB-infowindow-container {
   border-radius: 4px;
   overflow: hidden;
 }
-
-.Cdb-infowindow-inner, .Cdb-infowindow-header-bg {
-  padding: 20px 24px 18px 24px;
+.CDB-infowindow-inner,
+.CDB-infowindow-headerBg {
+  padding: 20px 24px 18px;
 }
-
-.Cdb-infowindow-inner--scroll {
-  padding-right: 12px;
-  position: relative;
+.CDB-infowindow-inner.CDB-infowindow-inner--scroll .CDB-infowindow-list {
+  max-height: 200px;
 }
-
 .is-header {
-  .Cdb-infowindow-header-bg {
+  .CDB-infowindow-headerBg {
     padding-bottom: 12px;
   }
-  .Cdb-infowindow-inner {
+  .CDB-infowindow-inner {
     padding-top: 12px;
   }
 }
-
-.Cdb-infowindow-subtitle {
-  font-size: $sFontSize-smaller;
+.CDB-infowindow-header .CDB-infowindow-tabs {
+  margin: -20px -24px 18px;
+}
+.CDB-infowindow-subtitle {
+  font-size: 9px;
   font-weight: $sFontWeight-lighter;
   line-height: $sLineHeight-smaller;
   text-transform: uppercase;
-} 
-
-.Cdb-infowindow-title {
+}
+.CDB-infowindow-title {
   font-size: $sFontSize-small;
   font-weight: $sFontWeight-lighter;
   line-height: $sLineHeight-small;
 }
-
-.is-scroll {
-  padding-bottom: 24px;
-  max-height: 200px;
-  overflow: scroll;
-  position: relative;
-  &:after {
-    content: "";
-    position: absolute;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    height: 32px;
-  }
+.CDB-infowindow-link {
+  width: 90%;
+  display: inline-block;
+  word-break: break-word;
+  // TODO: add style for the hovers
 }
 
-.Cdb-infowindow-list {
-  display: none;
+.CDB-infowindow-inner.has-scroll {
+  position: relative;
+  padding-right: 12px;
 }
 
-.Cdb-infowindow-list:first-child {
-  display: block;
+.has-scroll .CDB-infowindow-list {
+  position: relative;
+  max-height: 200px;
+  overflow: scroll;
 }
 
-.Cdb-infowindow-list-item {
+.CDB-infowindow-listItem {
   margin-top: 8px;
   &:first-child {
     margin-top: 0;
   }
 }
 
-.is-header .Cdb-infowindow-media-title {
+.CDB-infowindow-list .ps-scrollbar-y-rail {
+  width: 1px !important; // Resets standard scroll track color
+  opacity: 0.2 !important; // Resets standard scroll track color
```

Comment:
!important should not be used<br>Properties should be ordered width, margin-right, background, opacity, z-index